### ISSUE TITLE
Add Date Time Picker for QoL

### ISSUE THREAD

**Issue by HeriLFIU** [kytos-ng/maintenance#95]
When selecting a start and end date for a maintenance window it must be inputted in a specific format which I believe is a bit unintuitive and cumbersome, as seen below: 

![image](https://github.com/user-attachments/assets/6d3cc1e5-8bcf-4077-a4e2-f0ea78d4202c)

![image](https://github.com/user-attachments/assets/c9e9143e-6284-457b-b021-3443fb6bf63c)

This is why I believe that something like a datetime picker should be used to make this process simpler.

**Comment by HeriLFIU** [comment]
@viniarck Ya, that's what I was thinking, some like a datetime picker to make the selection of dates for `maintenance` easier.

**Comment by viniarck** [comment]
Right. Sounds good. We'll consider introducing a datetime picker in the `ui` in the future. Let's keep this in mind.
